recipes/mlinsights/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "mlinsights" %}
+{% set version = "0.3.543" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/sdpython/mlinsights/archive/refs/tags/v540.tar.gz  # Unfortunately the PyPI releases don't contain a license file and the GitHub tags don't match the PyPI releases
```

Comment:
It would also be fine to use the PyPI releases and just add the LICENSE manually to the feedstock.